src/DonationForms/Blocks/DonationFormBlock/resources/app/index.tsx

```diff
@@ -60,7 +59,7 @@ function DonationFormBlockApp({
     if (formFormat === 'modal' || formFormat === 'reveal') {
         return (
             <ModalForm
-                openFormButton={openFormButton}
+                buttonText={openFormButton}
```

Comment:
Let's add an `unreleased` tag in the top of this export to document this change.

Reply:
Resolved: ede7b291b3d574bf76f1d725674832a1e0b74c00